apps/communications/contacts/js/export/contacts_exporter.js

```diff
@@ -156,25 +154,14 @@ window.ContactsExporter = function ContactsExporter(theStrategy) {
   };
 
   //
-  // Based on the strategy configure the progress display to show a
-  // determinative or indeterminate ui depending on the strategy
-  //
-  var _configureProgress = function _configureProgress() {
-    determinativeProgress =
```

Comment:
ok, but one more change required here, you can leave line 29 unchanged but you shall update function **``_configureProgress()``**. It needs to comment existing lines 164 & 165,  change line 163 to ``determinativeProgress = true;`` and put a comment, which you made for line 29, before commented lines mentioning also that it's requirement for ResSquare project